src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java

```diff
@@ -586,6 +586,30 @@ public static int indexOfType(final Throwable throwable, final Class<? extends T
         return indexOf(throwable, type, fromIndex, true);
     }
 
+    /**
+     * Return whether the given throwable is a checked exception:
+     * that is, neither a RuntimeException nor an Error.
+     *
+     * <pre>
+     * ExceptionUtils.isCheckedException(new Exception())                    = true
+     * ExceptionUtils.isCheckedException(new SQLException())                 = true
+     * ExceptionUtils.isCheckedException(new RuntimeException())             = false
+     * ExceptionUtils.isCheckedException(new IllegalArgumentException(""))   = false
+     * ExceptionUtils.isCheckedException(new Throwable())                    = true
+     * ExceptionUtils.isCheckedException(null)                               = false
+     * </pre>
+     *
+     * @param throwable the throwable to check
+     * @return whether the throwable is a checked exception
+     * @see java.lang.Exception
+     * @see java.lang.RuntimeException
+     * @see java.lang.Error
+     * @since 3.12.1
```

Comment:
Adding a new API is not a maintenance version change per semantic versioning guidelines, the version should be the next minor one.

Reply:
HI @garydgregory 

Don't understand. what version should u put in order to maintain the guidelines?

Reply:
Hi @garydgregory 
Changed